### PR TITLE
Jellyfin 10.8.3

### LINKED ISSUE #5388: Update to Jellyfin 10.8.1 -> dependency of jellyfin-androidtv 0.14.0
### Is this a new Bug?

- [X] I checkd that the bug hasn't been reported before

### Package Name

Jellyfin

### Package Version

10.8.0

### Device Model

DS4XX

### Device Architecture

x86_64

### Firmware Version

DSM 7.0.1-42218 Update 2

### What happened?

Hi.  This is not so much a bug in the pacakage as it's a request to update it.

Android (TV) clients who are set to auto-update will not connect to JellyFin servers lower than 10.8.1 which is 1 pointversion ahead of the current package supported by SynoCommunity. 

Hi, @publicarray, @timrettop I would have notified stevenliuit but it looks like the @ doesn't pick him up yet. 

### Reproduction steps

```bash
1. Install jellyfin-androidtv 0.14.0 to your preffered device
2. Have the latest jellyfin package (10.8.0) on your DSM running
3. Open your jellyfin client on your android TV and discover server
4. Attempt to connect

Error message : "This server uses Jellyfin version XXX which is unsupported. Please update to jellyfind 10.8.1 to continue using this app.
```

### Install Log

```log
2021/10/31 17:26:14	install jellyfin 10.7.6-3 Begin preinst
2021/10/31 17:26:14	===> Step preinst. USER=sc-jellyfin GROUP=sc-media SHARE_PATH=
2021/10/31 17:26:14	install jellyfin 10.7.6-3 End preinst ret=[0]
2021/10/31 17:26:15	install jellyfin 10.7.6-3 Begin /bin/mkdir -p /volume1/@appstore/jellyfin
2021/10/31 17:26:15	install jellyfin 10.7.6-3 End /bin/mkdir -p /volume1/@appstore/jellyfin ret=[0]
2021/10/31 17:26:15	install jellyfin 10.7.6-3 Begin /bin/rm -rf /volume1/@appstore/jellyfin
2021/10/31 17:26:15	install jellyfin 10.7.6-3 End /bin/rm -rf /volume1/@appstore/jellyfin ret=[0]
2021/10/31 17:26:15	install jellyfin 10.7.6-3 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.bihAoX/package /volume1/@appstore/jellyfin
2021/10/31 17:26:15	install jellyfin 10.7.6-3 End /bin/mv -f /volume1/@tmp/pkginstall/extract.bihAoX/package /volume1/@appstore/jellyfin ret=[0]
2021/10/31 17:26:15	install jellyfin 10.7.6-3 Begin /bin/rm -rf /var/packages/jellyfin
2021/10/31 17:26:15	install jellyfin 10.7.6-3 End /bin/rm -rf /var/packages/jellyfin ret=[0]
2021/10/31 17:26:15	install jellyfin 10.7.6-3 Begin /bin/mkdir -p /var/packages/jellyfin
2021/10/31 17:26:15	install jellyfin 10.7.6-3 End /bin/mkdir -p /var/packages/jellyfin ret=[0]
2021/10/31 17:26:15	install jellyfin 10.7.6-3 Begin /bin/touch /var/packages/jellyfin/installing
2021/10/31 17:26:15	install jellyfin 10.7.6-3 End /bin/touch /var/packages/jellyfin/installing ret=[0]
2021/10/31 17:26:15	install jellyfin 10.7.6-3 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.bihAoX/INFO /var/packages/jellyfin/INFO
2021/10/31 17:26:15	install jellyfin 10.7.6-3 End /bin/mv -f /volume1/@tmp/pkginstall/extract.bihAoX/INFO /var/packages/jellyfin/INFO ret=[0]
2021/10/31 17:26:15	install jellyfin 10.7.6-3 Begin /bin/rm -rf /var/packages/jellyfin/scripts
2021/10/31 17:26:15	install jellyfin 10.7.6-3 End /bin/rm -rf /var/packages/jellyfin/scripts ret=[0]
2021/10/31 17:26:15	install jellyfin 10.7.6-3 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.bihAoX/scripts /var/packages/jellyfin/scripts
2021/10/31 17:26:15	install jellyfin 10.7.6-3 End /bin/mv -f /volume1/@tmp/pkginstall/extract.bihAoX/scripts /var/packages/jellyfin/scripts ret=[0]
2021/10/31 17:26:15	install jellyfin 10.7.6-3 Begin /bin/rm -rf /var/packages/jellyfin/WIZARD_UIFILES
2021/10/31 17:26:15	install jellyfin 10.7.6-3 End /bin/rm -rf /var/packages/jellyfin/WIZARD_UIFILES ret=[0]
2021/10/31 17:26:15	install jellyfin 10.7.6-3 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.bihAoX/WIZARD_UIFILES /var/packages/jellyfin/WIZARD_UIFILES
2021/10/31 17:26:15	install jellyfin 10.7.6-3 End /bin/mv -f /volume1/@tmp/pkginstall/extract.bihAoX/WIZARD_UIFILES /var/packages/jellyfin/WIZARD_UIFILES ret=[0]
2021/10/31 17:26:15	install jellyfin 10.7.6-3 Begin /bin/rm -rf /var/packages/jellyfin/conf
2021/10/31 17:26:15	install jellyfin 10.7.6-3 End /bin/rm -rf /var/packages/jellyfin/conf ret=[0]
2021/10/31 17:26:15	install jellyfin 10.7.6-3 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.bihAoX/conf /var/packages/jellyfin/conf
2021/10/31 17:26:15	install jellyfin 10.7.6-3 End /bin/mv -f /volume1/@tmp/pkginstall/extract.bihAoX/conf /var/packages/jellyfin/conf ret=[0]
2021/10/31 17:26:19	install jellyfin 10.7.6-3 Begin postinst
2021/10/31 17:26:19	===> Step postinst. USER=sc-jellyfin GROUP=sc-media SHARE_PATH=
2021/10/31 17:26:19	Begin service_postinst
2021/10/31 17:26:19	End service_postinst
2021/10/31 17:26:19	install jellyfin 10.7.6-3 End postinst ret=[0]
2021/10/31 17:26:20	install jellyfin 10.7.6-3 Begin start-stop-status start
2021/10/31 17:26:20	install jellyfin 10.7.6-3 End start-stop-status start ret=[0]
2022/01/22 16:48:40	upgrade jellyfin 10.7.6-3 Begin start-stop-status stop
2022/01/22 16:48:43	upgrade jellyfin 10.7.6-3 End start-stop-status stop ret=[0]
2022/01/22 16:48:45	upgrade jellyfin 10.7.6-3 Begin preupgrade
2022/01/22 16:48:45	===> Step preupgrade. USER=sc-jellyfin GROUP=sc-media SHARE_PATH=
2022/01/22 16:48:45	upgrade jellyfin 10.7.6-3 End preupgrade ret=[0]
2022/01/22 16:48:45	upgrade jellyfin 10.7.6-3 Begin preuninst
2022/01/22 16:48:45	===> Step preuninst. USER=sc-jellyfin GROUP=sc-media SHARE_PATH=
2022/01/22 16:48:45	upgrade jellyfin 10.7.6-3 End preuninst ret=[0]
2022/01/22 16:48:46	upgrade jellyfin 10.7.6-3 Begin /bin/rm -rf /volume1/@appstore/jellyfin
2022/01/22 16:48:46	upgrade jellyfin 10.7.6-3 End /bin/rm -rf /volume1/@appstore/jellyfin ret=[0]
2022/01/22 16:48:46	upgrade jellyfin 10.7.6-3 Begin /bin/rm -rf /volume1/@apptemp/jellyfin
2022/01/22 16:48:46	upgrade jellyfin 10.7.6-3 End /bin/rm -rf /volume1/@apptemp/jellyfin ret=[0]
2022/01/22 16:48:46	upgrade jellyfin 10.7.6-3 Begin postuninst
2022/01/22 16:48:46	===> Step postuninst. USER=sc-jellyfin GROUP=sc-media SHARE_PATH=
2022/01/22 16:48:46	upgrade jellyfin 10.7.6-3 End postuninst ret=[0]
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 Begin preinst
2022/01/22 16:48:47	===> Step preinst. USER=sc-jellyfin GROUP=sc-media SHARE_PATH=
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 End preinst ret=[0]
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 Begin /bin/mkdir -p /volume1/@appstore/jellyfin
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 End /bin/mkdir -p /volume1/@appstore/jellyfin ret=[0]
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 Begin /bin/rm -rf /volume1/@appstore/jellyfin
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 End /bin/rm -rf /volume1/@appstore/jellyfin ret=[0]
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.S8CrtU/package /volume1/@appstore/jellyfin
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 End /bin/mv -f /volume1/@tmp/pkginstall/extract.S8CrtU/package /volume1/@appstore/jellyfin ret=[0]
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 Begin /bin/rm -rf /var/packages/jellyfin
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 End /bin/rm -rf /var/packages/jellyfin ret=[0]
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 Begin /bin/mkdir -p /var/packages/jellyfin
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 End /bin/mkdir -p /var/packages/jellyfin ret=[0]
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 Begin /bin/touch /var/packages/jellyfin/installing
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 End /bin/touch /var/packages/jellyfin/installing ret=[0]
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.S8CrtU/INFO /var/packages/jellyfin/INFO
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 End /bin/mv -f /volume1/@tmp/pkginstall/extract.S8CrtU/INFO /var/packages/jellyfin/INFO ret=[0]
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 Begin /bin/rm -rf /var/packages/jellyfin/scripts
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 End /bin/rm -rf /var/packages/jellyfin/scripts ret=[0]
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.S8CrtU/scripts /var/packages/jellyfin/scripts
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 End /bin/mv -f /volume1/@tmp/pkginstall/extract.S8CrtU/scripts /var/packages/jellyfin/scripts ret=[0]
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 Begin /bin/rm -rf /var/packages/jellyfin/WIZARD_UIFILES
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 End /bin/rm -rf /var/packages/jellyfin/WIZARD_UIFILES ret=[0]
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.S8CrtU/WIZARD_UIFILES /var/packages/jellyfin/WIZARD_UIFILES
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 End /bin/mv -f /volume1/@tmp/pkginstall/extract.S8CrtU/WIZARD_UIFILES /var/packages/jellyfin/WIZARD_UIFILES ret=[0]
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 Begin /bin/rm -rf /var/packages/jellyfin/conf
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 End /bin/rm -rf /var/packages/jellyfin/conf ret=[0]
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.S8CrtU/conf /var/packages/jellyfin/conf
2022/01/22 16:48:47	upgrade jellyfin 10.7.7-4 End /bin/mv -f /volume1/@tmp/pkginstall/extract.S8CrtU/conf /var/packages/jellyfin/conf ret=[0]
2022/01/22 16:48:49	upgrade jellyfin 10.7.7-4 Begin postinst
2022/01/22 16:48:49	===> Step postinst. USER=sc-jellyfin GROUP=sc-media SHARE_PATH=
2022/01/22 16:48:49	Begin service_postinst
2022/01/22 16:48:49	End service_postinst
2022/01/22 16:48:49	upgrade jellyfin 10.7.7-4 End postinst ret=[0]
2022/01/22 16:48:49	upgrade jellyfin 10.7.7-4 Begin postupgrade
2022/01/22 16:48:49	===> Step postupgrade. USER=sc-jellyfin GROUP=sc-media SHARE_PATH=
2022/01/22 16:48:49	upgrade jellyfin 10.7.7-4 End postupgrade ret=[0]
2022/01/22 16:48:49	upgrade jellyfin 10.7.7-4 Begin start-stop-status start
2022/01/22 16:48:50	upgrade jellyfin 10.7.7-4 End start-stop-status start ret=[0]
2022/02/08 21:27:36	(system) trigger jellyfin 10.7.7-4 Begin start-stop-status stop
2022/02/08 21:27:42	(system) trigger jellyfin 10.7.7-4 End start-stop-status stop ret=[0]
2022/02/08 21:32:02	(system) trigger jellyfin 10.7.7-4 Begin start-stop-status start
2022/02/08 21:32:03	(system) trigger jellyfin 10.7.7-4 End start-stop-status start ret=[0]
2022/05/03 20:02:21	stop jellyfin 10.7.7-4 Begin start-stop-status stop
2022/05/03 20:02:23	stop jellyfin 10.7.7-4 End start-stop-status stop ret=[0]
2022/05/03 20:03:02	start jellyfin 10.7.7-4 Begin start-stop-status start
2022/05/03 20:03:03	start jellyfin 10.7.7-4 End start-stop-status start ret=[0]
2022/05/03 21:17:21	stop jellyfin 10.7.7-4 Begin start-stop-status stop
2022/05/03 21:17:22	stop jellyfin 10.7.7-4 End start-stop-status stop ret=[0]
2022/05/03 21:17:33	start jellyfin 10.7.7-4 Begin start-stop-status start
2022/05/03 21:17:33	start jellyfin 10.7.7-4 End start-stop-status start ret=[0]
2022/08/10 19:15:01	upgrade jellyfin 10.7.7-4 Begin start-stop-status stop
2022/08/10 19:15:03	upgrade jellyfin 10.7.7-4 End start-stop-status stop ret=[0]
2022/08/10 19:15:06	upgrade jellyfin 10.7.7-4 Begin preupgrade
2022/08/10 19:15:06	Begin reload_inst_variables
2022/08/10 19:15:06	End reload_inst_variables
2022/08/10 19:15:06	Begin initialize_variables
2022/08/10 19:15:06	End initialize_variables
2022/08/10 19:15:06	===> Step preupgrade. USER=sc-jellyfin GROUP=sc-media SHARE_PATH=
2022/08/10 19:15:06	upgrade jellyfin 10.7.7-4 End preupgrade ret=[0]
2022/08/10 19:15:06	upgrade jellyfin 10.7.7-4 Begin preuninst
2022/08/10 19:15:06	===> Step preuninst. USER=sc-jellyfin GROUP=sc-media SHARE_PATH=
2022/08/10 19:15:06	upgrade jellyfin 10.7.7-4 End preuninst ret=[0]
2022/08/10 19:15:06	upgrade jellyfin 10.7.7-4 Begin /bin/rm -rf /volume1/@appstore/jellyfin
2022/08/10 19:15:07	upgrade jellyfin 10.7.7-4 End /bin/rm -rf /volume1/@appstore/jellyfin ret=[0]
2022/08/10 19:15:07	upgrade jellyfin 10.7.7-4 Begin /bin/rm -rf /volume1/@apptemp/jellyfin
2022/08/10 19:15:07	upgrade jellyfin 10.7.7-4 End /bin/rm -rf /volume1/@apptemp/jellyfin ret=[0]
2022/08/10 19:15:07	upgrade jellyfin 10.7.7-4 Begin postuninst
2022/08/10 19:15:07	===> Step postuninst. USER=sc-jellyfin GROUP=sc-media SHARE_PATH=
2022/08/10 19:15:07	upgrade jellyfin 10.7.7-4 End postuninst ret=[0]
2022/08/10 19:15:07	upgrade jellyfin 10.8.0-5 Begin preinst
2022/08/10 19:15:07	Begin reload_inst_variables
2022/08/10 19:15:07	End reload_inst_variables
2022/08/10 19:15:07	Begin initialize_variables
2022/08/10 19:15:07	End initialize_variables
2022/08/10 19:15:07	===> Step preinst. USER=sc-jellyfin GROUP=sc-media SHARE_PATH=
2022/08/10 19:15:07	upgrade jellyfin 10.8.0-5 End preinst ret=[0]
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 Begin /bin/mkdir -p /volume1/@appstore/jellyfin
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 End /bin/mkdir -p /volume1/@appstore/jellyfin ret=[0]
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 Begin /bin/rm -rf /volume1/@appstore/jellyfin
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 End /bin/rm -rf /volume1/@appstore/jellyfin ret=[0]
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.aoOJzT/package /volume1/@appstore/jellyfin
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 End /bin/mv -f /volume1/@tmp/pkginstall/extract.aoOJzT/package /volume1/@appstore/jellyfin ret=[0]
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 Begin /bin/rm -rf /var/packages/jellyfin
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 End /bin/rm -rf /var/packages/jellyfin ret=[0]
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 Begin /bin/mkdir -p /var/packages/jellyfin
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 End /bin/mkdir -p /var/packages/jellyfin ret=[0]
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 Begin /bin/touch /var/packages/jellyfin/installing
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 End /bin/touch /var/packages/jellyfin/installing ret=[0]
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.aoOJzT/INFO /var/packages/jellyfin/INFO
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 End /bin/mv -f /volume1/@tmp/pkginstall/extract.aoOJzT/INFO /var/packages/jellyfin/INFO ret=[0]
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 Begin /bin/rm -rf /var/packages/jellyfin/scripts
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 End /bin/rm -rf /var/packages/jellyfin/scripts ret=[0]
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.aoOJzT/scripts /var/packages/jellyfin/scripts
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 End /bin/mv -f /volume1/@tmp/pkginstall/extract.aoOJzT/scripts /var/packages/jellyfin/scripts ret=[0]
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 Begin /bin/rm -rf /var/packages/jellyfin/WIZARD_UIFILES
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 End /bin/rm -rf /var/packages/jellyfin/WIZARD_UIFILES ret=[0]
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.aoOJzT/WIZARD_UIFILES /var/packages/jellyfin/WIZARD_UIFILES
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 End /bin/mv -f /volume1/@tmp/pkginstall/extract.aoOJzT/WIZARD_UIFILES /var/packages/jellyfin/WIZARD_UIFILES ret=[0]
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 Begin /bin/rm -rf /var/packages/jellyfin/conf
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 End /bin/rm -rf /var/packages/jellyfin/conf ret=[0]
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 Begin /bin/mv -f /volume1/@tmp/pkginstall/extract.aoOJzT/conf /var/packages/jellyfin/conf
2022/08/10 19:15:08	upgrade jellyfin 10.8.0-5 End /bin/mv -f /volume1/@tmp/pkginstall/extract.aoOJzT/conf /var/packages/jellyfin/conf ret=[0]
2022/08/10 19:15:10	upgrade jellyfin 10.8.0-5 Begin postinst
2022/08/10 19:15:10	Begin reload_inst_variables
2022/08/10 19:15:10	End reload_inst_variables
2022/08/10 19:15:10	Begin initialize_variables
2022/08/10 19:15:10	End initialize_variables
2022/08/10 19:15:10	===> Step postinst. USER=sc-jellyfin GROUP=sc-media SHARE_PATH=
2022/08/10 19:15:10	Begin save_wizard_variables
2022/08/10 19:15:10	End save_wizard_variables
2022/08/10 19:15:10	Begin syno_sync_var_folder
2022/08/10 19:15:10	End syno_sync_var_folder
2022/08/10 19:15:10	Begin service_postinst
2022/08/10 19:15:10	End service_postinst
2022/08/10 19:15:10	upgrade jellyfin 10.8.0-5 End postinst ret=[0]
2022/08/10 19:15:10	upgrade jellyfin 10.8.0-5 Begin postupgrade
2022/08/10 19:15:10	Begin reload_inst_variables
2022/08/10 19:15:10	End reload_inst_variables
2022/08/10 19:15:10	Begin initialize_variables
2022/08/10 19:15:10	End initialize_variables
2022/08/10 19:15:10	===> Step postupgrade. USER=sc-jellyfin GROUP=sc-media SHARE_PATH=
2022/08/10 19:15:10	Begin syno_sync_var_folder
2022/08/10 19:15:10	End syno_sync_var_folder
2022/08/10 19:15:10	upgrade jellyfin 10.8.0-5 End postupgrade ret=[0]
2022/08/10 19:15:10	upgrade jellyfin 10.8.0-5 Begin start-stop-status start
2022/08/10 19:15:11	upgrade jellyfin 10.8.0-5 End start-stop-status start ret=[0]
2022/08/10 19:16:35	(system) trigger jellyfin 10.8.0-5 Begin start-stop-status stop
2022/08/10 19:16:38	(system) trigger jellyfin 10.8.0-5 End start-stop-status stop ret=[0]
2022/08/10 19:16:45	(system) trigger jellyfin 10.8.0-5 Begin start-stop-status start
2022/08/10 19:16:45	(system) trigger jellyfin 10.8.0-5 End start-stop-status start ret=[0]
```

### Service Log

_No response_

### Other Logs

_No response_

### PR DESCRIPTION
**Pull request by stevenliuit**
## Description
update jellyfin to 10.8.3 version

Closes #5388

## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully
- [x] Package upgrade completed successfully (Manually install the package again)
- [ ] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Update-Policy#tests-checks)
- [ ] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

- [ ] Bug fix
- [ ] New Package
- [x] Package update
- [ ] Includes small framework changes
- [ ] This change requires a documentation update (e.g. Wiki)

### CODE REVIEW

**Comment by publicarray** [comment]
Thanks for continuing to update it. As we know now .net 6.0 requires more modern libraries from the host OS which means we either drop DSM6 support or use bubblewrap (see https://github.com/Servarr/spksrc/commits?author=ta264) to include a thin Debian environment with updated libs.

Thanks to @hgy59 and #5383 we also exclude specific arches e.g. comcerto. (Thanks @ta264 https://discord.com/channels/732558169863225384/905064261984845836/1004134443755241582)

So for now, please add the following to the makefile:

```
UNSUPPORTED_ARCHS += comcerto2k
REQUIRED_MIN_DSM = 7.0
```

File: spk/jellyfin/Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Remember to also update jellyfin-web
 SPK_NAME = jellyfin
-SPK_VERS = 10.8.0
-SPK_REV = 5
+SPK_VERS = 10.8.3
+SPK_REV = 7
```
Comment (publicarray) [inline spk/jellyfin/Makefile:4]: We are not so fast :)

```suggestion
SPK_REV = 6
```

**Comment by MothDoctor** [comment]
Will someone finally release this update, please? Can't use Jellyfin on Google Chromecast 4.0 since the client app already updated itself to 10.8.1, while this package which waits another 6 weeks months to be pushed, and PR is now closed for an unknown reason...

Jellyfin on NAS is awesome, but pretty useless if updating some versions numbers takes so much time :(

**Comment by hgy59** [comment]
@MothDoctor if you are in a hurry, you can download packages created by the github build action for manual installation.
(navigate: checks - build - artifacts)

**Comment by stevenliuit** [comment]
> @stevenliuit目前版本在 DSM 6.2.4 (aarch64) 的 DS218 上运行良好， 但在我的 DS218+ (x64 - apollolake) 上的 DSM 7.0.1 上无法正常运行 jellyfin 在以下几行之后记录。
> 
> ```
> [17:31:28] [INF] [1] Main: Startup complete 0:00:09.6917848
> [17:31:30] [INF] [10] Emby.Server.Implementations.ScheduledTasks.TaskManager: StartupTrigger fired for task: Update Plugins
> [17:31:30] [INF] [10] Emby.Server.Implementations.ScheduledTasks.TaskManager: Queuing task PluginUpdateTask
> [17:31:30] [INF] [12] Emby.Server.Implementations.ScheduledTasks.TaskManager: Executing Update Plugins
> [17:31:31] [INF] [8] Emby.Server.Implementations.ScheduledTasks.TaskManager: Update Plugins Completed after 0 minute(s) and 1 seconds
> [17:31:31] [INF] [8] Emby.Server.Implementations.ScheduledTasks.TaskManager: ExecuteQueuedTasks
> ```
> 
> Web UI 可以运行，但只是选择显示我的 Diskstation 的一个按钮并提供“服务器”，但无法连接。


> @stevenliuit current version runs fine on my DS218 with DSM 6.2.4 (aarch64) but it does not work on DSM 7.0.1 on my DS218+ (x64 - apollolake) jellyfin stops to log after the following lines.
> 
> ```
> [17:31:28] [INF] [1] Main: Startup complete 0:00:09.6917848
> [17:31:30] [INF] [10] Emby.Server.Implementations.ScheduledTasks.TaskManager: StartupTrigger fired for task: Update Plugins
> [17:31:30] [INF] [10] Emby.Server.Implementations.ScheduledTasks.TaskManager: Queuing task PluginUpdateTask
> [17:31:30] [INF] [12] Emby.Server.Implementations.ScheduledTasks.TaskManager: Executing Update Plugins
> [17:31:31] [INF] [8] Emby.Server.Implementations.ScheduledTasks.TaskManager: Update Plugins Completed after 0 minute(s) and 1 seconds
> [17:31:31] [INF] [8] Emby.Server.Implementations.ScheduledTasks.TaskManager: ExecuteQueuedTasks
> ```
> 
> The web UI runs, but shows only a button of my Diskstation and offers to "choose server", but it is not possible to connect.

please chose other bowser or clean this btowser cache

**Comment by hgy59** [comment]
> please chose other bowser or clean this btowser cache

@stevenliuit thanks mate! 👍

**Comment by hgy59** [comment]
@stevenliuit we need to include the current upstream master branch to get variable to dotnet (un) supported archs.
Can you rebase to upstream/master or merge upstream/master into this branch?

**Comment by stevenliuit** [comment]
> @stevenliuit we need to include the current upstream master branch to get variable to dotnet (un) supported archs.
> Can you rebase to upstream/master or merge upstream/master into this branch?

I'm on vacation recently, and I may not be able to use the computer for a week

**Comment by hgy59** [comment]
@stevenliuit enjoy your vacation! I have done the rebase and adjusted the unsupported archs.

@MothDoctor any feedback on the latest build is appreciated.

**Comment by hgy59** [comment]
> So for now, please add the following to the makefile:
> 
> ```
> UNSUPPORTED_ARCHS += comcerto2k
> REQUIRED_MIN_DSM = 7.0
> ```

@publicarray I run successfully this release on DS218 (aarch64) with `DSM 6.2.4-25556 Update 6`
I have tested music playback only. Is there any function that might not work on DSM < 7?

**Comment by ta264** [comment]
I'm no jellyin expert, but my experience with .NET and Radarr is that generally if it starts it's ok. The DSM 6 support seemed to vary by model, I guess depending on the particular kernel / glibc they came with. I get the sense newer glibc is needed for arm devices than x64 but I don't have any concrete minimum versions I'm afraid.

**Comment by MothDoctor** [comment]
> @MothDoctor if you are in a hurry, you can download packages created by the github build action for manual installation. (navigate: checks - build - artifacts)

Thanks! It works fine at DSM 7.1 on DS720+ hardware :)
I hope this gonna get updated online for other folks :)

**Comment by azmi-ah** [comment]
Hi! I am looking for some help, please. Jellyfin is installed on my Synology NAS. Recently, I was suddenly not able to access the login page of my Jellyfin server through the IOS devices (App and Safari). It is diverting to the setup page, although my admin user is already set and Jellyfin was already configured and being used for many months. Any idea how to solve this issue? The issue started, when I was traveling. I used to access Jellyfin through the dns address created by TP-Link router. The access to the login page was working for a few days, then it didn’t. So, I tried accessing it from the VPN (OpenVPN), but I couldn't. When I came back to my local network, I was able to login for a few days on my laptop (Chrome), but still couldn't access the login page on my iPhone or iPad. Now, I am not able to access the login page even from the laptop (Chrome). Before the login through Chrome became inaccessible, I updated to 10.8.3-6, which is the latest update (in Synology). Jellyfin is installed in Synology DSM 6.2.4-25556 Update 6 as a Package (not docker). One more note, while troubleshooting, I noticed that jellyfin-web file is not available in the "/volume1/@appstore/jellyfin/share" folder. Any idea how to resolve this matter? Thank you.

**Pull request merged**